Derikka/CavityTest_code.py

```diff
@@ -0,0 +1,68 @@
+from pydm import Display
+from PyQt5 import QtGui, QtCore, QtWidgets
+import epics
+from epics import caget, caput
+
+class CavityTest(Display):
+
+	def ui_filename(self):
+		return 'CavityTest.ui'	
+
+	def __init__(self, parent=None, args=None):
+		super(CavityTest, self).__init__(parent=parent, args=args)
+
+		# Parameters
+		self.PV = ''									# current PV
```

Comment:
I would stay away from inline comments (also the comment is really far to the right for some reason?). I would just do something like

`# Parameters

# current PV
self.PV = ''`